docs/installation/deployment/single_server.rst

```diff
@@ -39,9 +39,27 @@ A server
 
 Ensure you have a server with ``root`` privileges. We assume you can directly
 ssh to the machine as ``root`` user. If that's not the case, a user with
-``sudo`` will also work. Python 3 must be available on the server. Debian 9/10
-are officially supported operating systems, though it is likely the
-installation also works on Ubuntu. CentOS/RedHat *might* work.
+``sudo`` will also work. Python 3 must be available on the server.
+
+.. note:: Make sure there is enough space in ``/var/lib/docker``. You need at
+   least 8 GB to download all Docker container images.
+
+**Supported operating systems**
+
+Support for different Linux flavours is maintained in the `Ansible collection`_ used
+for deployment.
+
+Currently the following OS flavours are supported:
+
+- Debian: buster (10, actively supported), stretch (9, actively supported), jessie (8)
+- Ubuntu: eoan (EOL), disco (EOL), cosmic (EOL), bionic (18.04 LTS). focal (20.04 LTS)
+  is not tested yet
+- SUSE Enterprise Linux: 15 (actively supported)
+- OpenSUSE: 15.1
+- Red Hat: 7, 8
+- CentOS: 7 (actively supported), 8
```

Comment:
Maybe mark CentOS 8 as actively supported since its tested

Reply:
I have limited Ansible knowledge, we use Chef but for deploying Open-zaak just the gitlab ci/cd.